pkg/agent/openflow/client.go

```diff
@@ -312,6 +315,43 @@ func (c *client) addFlows(cache *flowCategoryCache, flowCacheKey string, flows [
 	return nil
 }
 
+func (c *client) hardModifyFlows(cache *flowCategoryCache, flowCacheKey string, flows []binding.Flow) error {
+	oldFCacheI, ok := cache.Load(flowCacheKey)
```

Comment:
nit: feel not neat to have a shorthand in the middle of a name. The function uses "flow" in many places, just "oldFlowCacheI"?

Reply:
Updated. Also `oldFlowCache`